### DISCUSSION TITLE
cant get report alert to slack

### DISCUSSION THREAD

**Discussion by galfrylich** [aquasecurity/trivy-operator#1396] [Help & Support]
Hello, im trying to set the webhooks with slack, added the url for my channel and im getting this from slack channel 
"Due to a high volume of activity, we are not displaying some messages from this incoming webhook integration."

what am i doing wrong?

**Comment by chen-keinan** [comment]
@galfrylich are you using [postee](https://github.com/aquasecurity/postee) ?

**Reply by galfrylich** [reply]
yes just install it and added the config file in the ui, inside the template operator slack 

this is the config file :
routes:
- name: Trivy Operator Alerts
  input: input.report.summary.criticalCount > 0 # You can customize this based on your needs
  actions: [send-slack-msg]
  template: trivy-operator-slack

# Templates are used to format a message
templates:
- name: trivy-operator-slack
  rego-package: postee.trivyoperator.slack

# Actions are target services that should consume the messages
actions:
- name: send-slack-msg
  type: slack
  enable: true
  url: <slack webhook url>

**Reply by galfrylich** [reply]
"open rego-templates/custom/lost+found: permission denied "
now this is ny error

**Reply by chen-keinan** [reply]
@galfrylich can you pass this input in `trivy-operator` slack channel maybe someone from community has a similar issue
